scripts/vm-util.py

```diff
@@ -346,7 +330,6 @@ def do_qemu(args):
 qemu_parser.add_argument("--no-kvm", action="store_true")
 qemu_parser.add_argument("--virtual-cpu", action="store_true")
 qemu_parser.add_argument("--no-smp", action="store_true")
-qemu_parser.add_argument("--virtual-boot", action="store_true")
```

Comment:
Can't we do `--virtual-boot` with MB2?

Reply:
qemu's `-kernel` only supports multiboot1.

Reply:
Uh, that's annoying. That might be a reason to keep MB1 support.

Reply:
The problem with multiboot1 support is that it will, by definition, never give us a RSDP when booting in UEFI-mode. It only works with BIOS, and maybe CSM (not sure about that).

Reply:
FWIW it also supports the Linux boot protocol IIRC

it might also be relatively easy to add mb2 support

Reply:
I guess we could also just set up EFI vars to boot `eir-uefi`?